DownloadManager/Plugin.DownloadManager.Android/DownloadFileImplementation.cs

```diff
@@ -92,20 +94,44 @@ public DownloadFileImplementation (ICursor cursor)
             Url = cursor.GetString (cursor.GetColumnIndex (Android.App.DownloadManager.ColumnUri));
         }
 
-        public void StartDownload (Android.App.DownloadManager downloadManager, string destinationPathName)
+        private bool _mobileNetworkAllowed = true;
+
+        public bool MobileNetworkAllowed
         {
-            using (var downloadUrl = Uri.Parse (Url))
-            using (var request = new Android.App.DownloadManager.Request (downloadUrl)) {
+            get
+            {
+                return _mobileNetworkAllowed;
+            }
+            set
+            {
+                _mobileNetworkAllowed = value;
+                if (Request != null && value == false) {
+                    Request.SetAllowedNetworkTypes(Android.App.DownloadNetwork.Wifi);
```

Comment:
The AndroidUseLatestPlatformSdk overrides the TargetFrameworkVersion tag i think. The problem is that if you set it to use the latest Platform SDK it sets AndroidUseLatestPlatformSdk but leaves the TargetFrameworkVersion as it is. It can therefore be considered. As @SimonSimCity pointed out, we should try to keep the SDK platform version low since we are **creating** and Android library.

The Target Framework for Plugin.DownloadMnager.Android is now Android 4.1 (API level 16), and the Example uses the latest Android SDK possible.

Example using latest possible SDK
https://github.com/SimonSimCity/Xamarin-CrossDownloadManager/blob/master/Sample/Droid/DownloadExample.Droid.csproj#L18

Upgraded to version 4.1
https://github.com/fela98/Xamarin-CrossDownloadManager/blob/AllowMobileNetwork/DownloadManager/Plugin.DownloadManager.Android/Plugin.DownloadManager.Android.csproj#L19